app/models/nic/bond.rb

```diff
@@ -38,12 +19,8 @@ def self.humanized_name
 
     private
 
-    def ensure_virtual
-      self.virtual = true
-    end
-
     def enc_attributes
-      @enc_attributes ||= (super + %w(mode attached_devices bond_options))
+      @enc_attributes ||= (super + %w(mode bond_options))
```

Comment:
@ares that "super" stuff will be broken if we use the include instead of the parent class approach, isn't it? what do you see as a solution?